docs/changes.rst

```diff
@@ -563,6 +563,14 @@ Released: not yet
 
 * Test: Converted WBEMListener tests from unittest to pytest. (See issue #2179)
 
+* Enhance FakeWBEMConnection to allow user-defined providers for specific
+  WBEM request operations.  This allows user-defined providers for selected
+  instance requests (CreateInstance, ModifyInstance, DeleteInstance) and for
+  the InvokeMethod.  Includes the capability to register these providers with
+  a method `register_provider` in Faked_WBEMConnection.  This also creates
+  a CIM_Namespace provider to handle the CIM_Namespace class in the interop
+  namespace.  See issue #2062)
+
```

Comment:
We also should add an entry in the incompatible changes section, because this PR removes the old way of having method callbacks (`add_method_callback()` method and `methods` property on `FakedWBEMConnection` class).

Reply:
DONE